client/my-sites/stats/subscribers-section/index.tsx

```diff
@@ -52,13 +60,26 @@ export default function SubscribersSection( { siteId }: { siteId: string } ) {
 
 	return (
 		<div className="subscribers-section">
-			<div className="subscribers-section-heading">
-				<h1 className="highlight-cards-heading">Subscribers</h1>
+			{ /* TODO: Remove highlight-cards class and use a highlight cards heading component instead. */ }
+			<div className="subscribers-section-heading highlight-cards">
+				<h1 className="highlight-cards-heading">
+					{ translate( 'Subscribers' ) }{ ' ' }
+					<small>
+						<a
+							className="highlight-cards-heading-wrapper"
+							href={ '/people/subscribers/' + siteSlug }
+						>
+							{ translate( 'View all subscribers' ) }
+						</a>
+					</small>
+				</h1>
 				<div className="subscribers-section-legend" ref={ legendRef }></div>
 			</div>
 			{ isLoading && <StatsModulePlaceholder className="is-chart" isLoading /> }
 			{ ! isLoading && chartData.length === 0 && (
-				<p className="subscribers-section__no-data">No data availble for the specified period.</p>
+				<p className="subscribers-section__no-data">
+					{ translate( 'No data availble for the specified period.' ) }
```

Comment:
There is a typo in this string. `availble` --> `available`

Reply:
Hi Emily, thank you for the heads up. We'll have this fixed in a follow-up PR shortly.

Reply:
I have the fix for this here: https://github.com/Automattic/wp-calypso/pull/75797.